Makefile

```diff
@@ -391,22 +365,22 @@ bin/konvoy-image-arm64:
 	ln -sf ../dist/konvoy-image_linux_arm64/konvoy-image bin/konvoy-image-arm64
 
 konvoy-image-linux:
-	$(MAKE) devkit.run GOOS=linux GOARCH=$(BUILDARCH) WHAT="make bin/konvoy-image"
-	$(MAKE) devkit.run GOOS=linux GOARCH=$(BUILDARCH) WHAT="make bin/konvoy-image-$(BUILDARCH)"
+	$(MAKE) make bin/konvoy-image GOOS=linux GOARCH=$(BUILDARCH)
+	$(MAKE) make bin/konvoy-image-$(BUILDARCH) GOOS=linux GOARCH=$(BUILDARCH)
```

Comment:
lets remove `$(MAKE) make bin/konvoy-image-$(BUILDARCH) GOOS=linux GOARCH=$(BUILDARCH)` it might be duplicate. try removing `bin/konvoy-image-amd64` and `bin/konvoy-image-arm64` too

Reply:
simplify `$(MAKE) make bin/konvoy-image-$(BUILDARCH) GOOS=linux GOARCH=$(BUILDARCH)` the GOOS=linux needs to hardcoded because this binary will be added to the Linux container image.

Reply:
check if `konvoy-image-linux` can be removed

Reply:
`konvoy-image-linux` removed.